src/objects/token/admin.py

```diff
@@ -104,7 +127,27 @@ def add_view(self, request, form_url="", extra_context=None):
 class PermissionInline(EditInlineAdminMixin, admin.TabularInline):
     model = Permission
     fk_name = "token_auth"
-    fields = ("object_type", "mode", "use_fields", "fields")
+    fields = (
+        "object_type",
+        "mode",
+        "use_fields",
+        "fields",
+        "get_uuid",
+        "get_version",
```

Comment:
I think it's confusing, because it looks like the permission works only for this version. In fact any permission works for all objecttype versions. 

@joeribekker The same comment - objecttype versions are not taken into account in permissions, so I don't know if displaying last version would be of any use. 
Do you still want it?

Reply:
nop, will respecc the ticket.